apps/src/appsApi.js

```diff
@@ -45,31 +56,53 @@ var ApiRequestHelper = function (baseUrl) {
   this.apiBaseUrl_ = baseUrl;
 };
 
+/**
+ * @param {!string} localUrl - API endpoint relative to API base URL.
+ * @param {!NodeStyleCallback} callback
+ * @param {*} failureValue - what to pass as the callback result if an error
+ *        occurs.
+ */
 ApiRequestHelper.prototype.get = function (localUrl, callback, failureValue) {
   $.ajax({
     url: this.apiBaseUrl_ + localUrl,
     type: 'get',
     dataType: 'json'
-  }).done(function (result /*, text*/) {
-    callback(result);
-  }).fail(function (/*request, status, error*/) {
-    callback(failureValue);
+  }).done(function (data /*, textStatus, jqXHR*/) {
+    callback(null, data);
+  }).fail(function (jqXHR, textStatus, errorThrown) {
+    callback(
+        new Error('textStatus: ' + textStatus + '; errorThrown: ' + errorThrown),
+        failureValue);
```

Comment:
Cool. In general, I think it's good to avoid depending on the result in the callback to be a particular value when we have a non-null error.